src/main/steps/c100-rebuild/check-your-answers/content.ts

```diff
@@ -68,6 +68,8 @@ export const enContent = {
   continue: 'Accept and continue',
   Yes: 'Yes',
   No: 'No',
+  yes: 'Yes',
+  no: 'No',
   'Dont know': 'Dont know',
```

Comment:
@sealDebolina I glanced through PR I think there are few misses when compared to PR 841
`. email: 'Email',
    dontKnowTheirEmailAddress: 'I dont know their email address',
    telephoneNumber: 'Telephone number',
    dontKnowTheirTelephoneNumber: 'I dont know their telephone number',`

Reply:
Hi Channabasappagouda Patil, thanks, That is not needed as there is a separate object(personalDetails) for context. we can use that